Readme.md

```diff
@@ -3,22 +3,41 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T237950)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [CellEnabledConverter.cs](./CS/WpfApplication241/CellEnabledConverter.cs) (VB: [CellEnabledConverter.vb](./VB/WpfApplication241/CellEnabledConverter.vb))
-* [DateEditCalendarEx.cs](./CS/WpfApplication241/DateEditCalendarEx.cs) (VB: [DateEditCalendarEx.vb](./VB/WpfApplication241/DateEditCalendarEx.vb))
+# WPF DateEdit - Disable Dates in the Drop-down Calendar
+
+This example forbids users to select specified days in the [DateEdit](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.DateEdit)'s drop-down calendar. The [DateEdit.MinValue](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.DateEdit.MinValue) and [DateEdit.MaxValue](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.DateEdit.MaxValue) properties allow you to specify the available date range.

```

Comment:
```suggestion
This example prevents users from selecting specific days in the [DateEdit](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.DateEdit)'s drop-down calendar. The [DateEdit.MinValue](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.DateEdit.MinValue) and [DateEdit.MaxValue](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.DateEdit.MaxValue) properties allow you to specify the available date range.
```

Reply:
Also, is "days" correct (Monday, Tuesday, Wednesday, etc.)? Or do you mean "dates"?